code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1161,8 +1161,11 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Neurotoxin"
 	glass_desc = "A drink that is guaranteed to knock you silly."
 
+/datum/reagent/consumable/ethanol/neurotoxin/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
```

Comment:
This is the wrong proc for this, I'm not sure what the right one is called. on_mob_added() or something. Check holder.dm